_templates/header.php

```diff
@@ -100,6 +100,7 @@
         <script>
             <?php include __DIR__.'/../scripts/jql.min.js'; ?>
             jQl.loadjQ('https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.0/jquery.min.js')
+            jQl.loadjQdep('https://cdnjs.cloudflare.com/ajax/libs/flipclock/0.7.8/flipclock.min.js')
```

Comment:
I've made a pull request for flipclock to be auto-updated on JSDelivr: https://github.com/jsdelivr/jsdelivr/pull/13425